streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/AssignorConfiguration.java

```diff
@@ -291,11 +271,7 @@ public String userEndPoint() {
         }
     }
 
-    public Admin adminClient() {
-        return adminClient;
-    }
-
-    public InternalTopicManager internalTopicManager() {
+    public InternalTopicManager internalTopicManager(final Admin adminClient) {
```

Comment:
Instead of passing the `Admin` as parameter, we could call `taskManager#adminClient()` in the next line, however, this requires to make the method `public` (it's package-private atm) thus I opted to for parameter-passing instead.